tribe-ext-extension-template.php

```diff
@@ -56,6 +56,20 @@ class Main extends Tribe__Extension {
 		 */
 		public $ecp_active = false;
 
+		private $settings;
+
+		/**
+		 * Set up if it only works with a specific view
+		 * Possible values
+		 * 'V1 legacy'
+		 * 'V2 updated'
+		 *
+		 * @return string
+		 *
+		 * @todo Remove this and corresponding method v2_views_check() if extension works with both V1 and V2 views.
+		 */
+		public $required_view = "V1 legacy";
```

Comment:
single quotes, not double

Reply:
why so lengthy? why not just `1` / `2` or `v1` / `v2` (`v` or `V`)?

Reply:
or maybe make it an array of views it _does_ work with?

Reply:
I used a full string because that string is then used in the warning message. I'll think about this some more.